packages/eslint-plugin/rules/i18n-hyphenated-range.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Internal dependencies
+ */
+const {
+	TRANSLATION_FUNCTIONS,
+	getTextContentFromNode,
+	getTranslateFunctionName,
+	getTranslateFunctionArgs,
+} = require( '../utils' );
+
+const EN_DASH = '–';
+const HYPHEN_IN_RANGE = /(\d\s+-\s+\d)|(\d-\d)/g;
+
+function replaceHyphenWithEnDash( string ) {
+	if ( string.match( HYPHEN_IN_RANGE ) ) {
+		return string.replace( '-', EN_DASH );
+	}
+}
+
+// see eslint-plugin-wpcalypso.
```

Comment:
Not sure what means this comment is, is this rule somehow related to `eslint-plugin-wpcalypso`?

Reply:
This was from copying the ellipsis rule as a template, I'll remove it.